adapter/config/default_config.go

```diff
@@ -58,6 +58,9 @@ var defaultConfig = &Config{
 			Location: "/home/wso2/security/truststore",
 		},
 		ArtifactsDirectory: "/home/wso2/artifacts",
+		SourceControl: sourceControl{
+			Enabled: false,
```

Comment:
We need other default configs here.  We can give default values for the below.

	Enabled bool
	// PollInterval how frequently the source watcher should be polled to get updates from the remote repository (in seconds)
	PollInterval int
	// RetryInterval how frequently the source watcher should retry to fetching artifacts from the remote repository (in seconds)
	RetryInterval int
	// ArtifactsDirectory is the FilePath where the api artifacts are created when fetched from the remote repository
	ArtifactsDirectory string

Reply:
Fixed with ca229b8